### ISSUE TITLE
Problem with one of the window we can move (can click)

### ISSUE THREAD

**Issue by Hermonii** [teelolws/EditModeExpanded#112]
Hi,
Since one of the last update, found that we cant click on anything at the same level of the "talking head"(i hope it's the same name, the box that pop when npc talk).
Hard to explain it in english, so may be screen will help : 

![image](https://github.com/teelolws/EditModeExpanded/assets/118864327/42231d56-92c2-4a9a-bb8b-f9062bca9180)

This one. If there a button like to close a window at the same place, i cant clic.
I moved it to an other place and that blocked others buttons.

And really weird (for me^^ i dont understand), looks like that sometime open and close the edite menu solve this problem for a while.
I tested with only this add on and with none. Bug is with the add.

Thanks in advance^^

**Comment by teelolws** [comment]
Can you show me an example? Show me which button its blocking.

**Comment by Hermonii** [comment]
Looks like it's all things that we can interact with. Like if the window was above all interface but hide.
At start i had this at the bottom and couldnt clic to craft something. I moved in top of my screen to test, i couldnt clic on the "x".

But really looks like it's all ui because i tried to reset the place (that place it just above actionsbutton), that block them too.

I made this to try to explain it better. 
https://imgur.com/qFtImRp

I could clic arround but not where this windows is (i show it just after). When i close the edit menu that solve the problem but isnt pemanent. Reloading or reloging make this problem again. Even with all others add on removed.

Hopping that will help. For the moment i moved it somewhere that cant block important stuffs (was above channel too, couldnt clic on item links^^').
